### PR TITLE
Connect to MetaMask using wagmi

### LINKED ISSUE #7: Top page
state | action | design
-- | -- | --
before connect wallet | NA | https://www.figma.com/file/acFBUHdVMIjUYkE4omv8LU/Chakra-UI-Figma-Kit-(Community)?node-id=807%3A9205
  | connect wallet |

### PR DESCRIPTION
**Pull request by alecrem**
Closes #7 

Sorry for the "no message" commits. I forgot that those rebasing steps do show on history.

### CODE REVIEW

File: components/wallet.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <>
+        {/* <Text mt={5}>{ account.address }</Text> */}
+        <Button mt={10} onClick={() => disconnect()}>Disconnect</Button>
+      </>
+    )
+  }
+
+  return (
+    <>
+      {connectors.map((connector) => (
```
[the remark this answers is not in the document]
Comment (alecrem) [reply components/wallet.tsx]: Ooooh! Sorry. I missed the link and only looked at the suggested change.

That was embarrassing... Thank you for your patience!

**Comment by alecrem** [comment]
The official example changed since I started doing this. [This is the version I looked at](https://github.com/tmm/wagmi/tree/2df3edee3b0dca446c91603b60b89f6833c0af16/examples/next/src/components).

**Comment by alecrem** [comment]
My understanding is that #7 just needs to be able to connect to Metamask for now.

The "Henkaku Membershit NFT" and "Badge Store" sections will be added to the front page when they are ready.

**Comment by hourglasshoro** [comment]
I'm confused myself, are we going to be the ones to go with MetaMaskConnector?

**Comment by SamuraiT** [comment]
connecting wallet was the one of the bottle neck to code approve function and another.
so I PRed and merged already as you know: https://github.com/henkaku-center/membership-nft-interface/pull/18
now, you can go for generateImageURL implementation which is heavier than this.
so I want you to focus on it for now not the top page.

**Pull request closed without merging**